docker/setup_configuration/data.yml

```diff
@@ -0,0 +1,25 @@
+sites_config_enable: true
+sites_config:
+  items:
+  - domain: example.com
+    name: Example site
+
+zgw_consumers_config_enable: true
+zgw_consumers:
+  services:
+  - identifier: notifications-api
+    label: Notificaties API
+    api_root: http://notificaties.local/api/v1/
```

Comment:
I'd add the service with such name to docker-compose so you would have an example of working configuration of ON and OZ